### DISCUSSION TITLE
@formily/antd 的国际化 如何实现？

### DISCUSSION THREAD

**Discussion by jackchenl** [alibaba/formily#2985] [Q&A 2.x]
你好，我在全局配置了antd的国际化为中文，正常使用antd的组件是显示中文的日期格式的，但现在使用 @formily/antd 的 DatePicker组件，还是英文的日期格式，请问如何配置？谢谢了

**Comment by cxhcmc** [comment]
遇到同样的问题，在ant design pro 中 如何结合i18n 实现多语言切换？

**Comment by elvis-KF** [comment]
+1 而且注入到DatePicker的locale属性也不生效

**Comment by ZeroJsus** [comment]
有没有人处理了 分享一下呢

**Comment by jackchenl** [comment]
已经解决了，antd 的加载需要和 @formily/antd 一样 具体参考：https://github.com/alibaba/formily/issues/1830
